Server/Utilities/CBXDevice.m

```diff
@@ -244,14 +244,19 @@ - (NSDictionary *)formFactorMap {
       @"iPhone9,2" : @"iphone 6+",
       @"iPhone9,4" : @"iphone 6+",
 
-      // iPhone 8/8+/X - derived from Simulator
+      // iPhone 8/8+/X
       @"iPhone10,1" : @"iphone 6",
       @"iPhone10,4" : @"iphone 6",
       @"iPhone10,5" : @"iphone 6+",
       @"iPhone10,2" : @"iphone 6+",
       @"iPhone10,3" : @"iphone 10",
       @"iPhone10,6" : @"iphone 10",
 
+      // iPhone XS/XS Max/XR - derived from Simulator
+      @"iPhone11,2" : @"iPhone11,2",
+      @"iPhone11,4" : @"iPhone11,4",
+      @"iPhone11,8" : @"iPhone11,8",
```

Comment:
We are mapping _from_ a model number _to_ a form factor.

The form factor will be one of:

"iphone 10", "iphone 10s max", or "iphone 10r"

Reply:
@jmoody *changed*